src/openmrs-backend-dependcies.ts

```diff
@@ -0,0 +1,49 @@
+import { openmrsFetch } from "@openmrs/esm-api";
+const semver = require("semver");
+import backEndDeps from "../backEndDeps.json";
+
+export function fetchBackendDeps() {
+  return openmrsFetch(`/ws/rest/v1/module?v=custom:(uuid,version)`, {
+    method: "GET"
+  });
+}
+
+export function checkBackendDeps(dependency: any) {
+  fetchBackendDeps().then(({ data }) => {
```

Comment:
Instead of fetching the backend deps every time that a systemjs module is loaded, we should just load the modules once upfront and then reuse the data every time that a systemjs module is loaded.